sapi/fpm/www.conf.in

```diff
@@ -295,7 +295,7 @@ pm.max_spare_servers = 3
 ;      it can accept the following format:
 ;      - %{seconds}d (default)
 ;      - %{milliseconds}d
-;      - %{mili}d
+;      - %{milli}d
```

Comment:
Is it ok not to mention the "deprecated" format here?
Should it warn about "old versions may not support this flavor"?